src/documents/adding.md

```diff
@@ -279,17 +194,321 @@ Your flying car has an auto-pilot feature, and you want to add a preference for
             "speakText": {
                 // some content not shown
             },
-            "autoPilot": {
-                "type": "gpii.firstDiscovery.autoPilot",
+            "autopilot": {
+                "type": "gpii.firstDiscovery.autopilot",
                 "panel": {
-                    "type": "gpii.firstDiscovery.panel.autoPilot",
-                    "container": ".gpiic-fd-prefsEditor-panel-autoPilot",
+                    "type": "gpii.firstDiscovery.panel.autopilot",
+                    "container": ".gpiic-fd-prefsEditor-panel-autopilot",
                     "template": "%templatePrefix/yesNo.html",
-                    "message": "%messagePrefix/autoPilot.json"
+                    "message": "%messagePrefix/autopilot.json"
                 }
             },
             // more content not shown
         }
     });
     ```
+    
+    At this point, you should have a new panel in the tool with a new preference called "Autopilot" that follows the "yesNo" template, and uses the text you specified in the message files. Changing the language setting should use the alternate the text in the panel with the corresponding language selection. 	
+
+    Now that you have a working panel and preference, you'll want to have that preference appear in on the confirm page so that the user can make sure that your new preference is set how they want it.
+     
+6. Edit `src/js.panels.js` to add the selectors for the "Autopilot" preference. 
+
+    A. Find the `"fluid.defaults()"` call for the confirm panel, which calls `"gpii.firstDiscovery.panel.confirm"`.
```

Comment:
How about "Find the component definition for gpii.firstDiscovery.panel.confirm" ?